The com_google_googleapis dependency calls the protobuf_deps function which isn't there yet for the
system protobuf. So add a dummy which does nothing which is enough for this use case.

Author: Alexander Grund (TU Dresden)

diff --git a/tensorflow/workspace.bzl b/tensorflow/workspace.bzl
index a6ea0094dde..eb7dba4ce56 100755
--- a/tensorflow/workspace.bzl
+++ b/tensorflow/workspace.bzl
@@ -585,6 +585,7 @@ def tf_repositories(path_prefix = "", tf_repo_name = ""):
         system_build_file = clean_dep("//third_party/systemlibs:protobuf.BUILD"),
         system_link_files = {
             "//third_party/systemlibs:protobuf.bzl": "protobuf.bzl",
+            "//third_party/systemlibs:protobuf_deps.bzl": "protobuf_deps.bzl",
         },
         urls = [
             "https://storage.googleapis.com/mirror.tensorflow.org/github.com/protocolbuffers/protobuf/archive/v3.9.2.zip",
diff --git a/third_party/systemlibs/protobuf_deps.bzl b/third_party/systemlibs/protobuf_deps.bzl
new file mode 100644
index 00000000000..8699b840ed4
--- /dev/null
+++ b/third_party/systemlibs/protobuf_deps.bzl
@@ -0,0 +1,4 @@
+"""Stub version of @com_google_protobuf//:protobuf_deps.bzl necessary for TF system libs"""
+
+def protobuf_deps():
+    pass
